audit/pom.xml

```diff
@@ -30,6 +30,14 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>gov.cms.ab2d</groupId>
+            <artifactId>common</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+            <type>test-jar</type>
+            <scope>test</scope>
```

Comment:
I think it might be a bit bit odd that the `type` is `test-jar` and the `scope` is `test`.
What happens if we remove these two attributes from here? Will it not work?

Reply:
It's only for test purposes, we wouldn't want this to be used for production. There's a JAR that contains reusable components for data setup in test.

Reply:
Sounds good.